shopinvader/services/service.py

```diff
@@ -98,6 +98,24 @@ def _get(self, _id):
     def _get_base_search_domain(self):
         return []
 
+    def _get_selection_label(self, record, field):
```

Comment:
can't you use `_convert_to_export`?
Example https://github.com/shopinvader/odoo-shopinvader/blob/12.0/shopinvader/models/res_partner.py#L129